src/pages/Home.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 import { useQuery } from 'react-query'
 import  Whoops404  from './Error';
+import { Link } from 'react-router-dom';
 
 const fetchProducts = async () => {
-  const res = await fetch(`https://dummyjson.com/products`)
-    return res.json()   
-}
-  
+  const response = await (await fetch(`https://dummyjson.com/products`)).json();
+  console.log('response1', response);
+  return response;
+};
+
 const Home = () => {
-  const {products, status} = useQuery('products', fetchProducts);
-  console.log(products);
+  const { data, status } = useQuery(
+    'products',
+    fetchProducts
+  );
+  const products = data?.products;
```

Comment:
by default react-query is returning the response as `data` so i added the extra line to assign the results to `products`